packages/api/src/schema/schema.yaml

```diff
@@ -1795,6 +2145,42 @@ paths:
               schema:
                 $ref: "#/components/schemas/error"
 
+  /vod-asset:
+    tags:
+      - vod-asset
+    get:
+      description: gets a list of vod assets defined by the user
+      responses:
+        200:
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/vod-asset"
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+    post:
+      description: creates a new vod asset
+      responses:
+        200:
```

Comment:
POST should respond with `201 Created` status ^_^

Reply:
Thanks for the catch!